### PR TITLE
CNV-25574: Release note: NEW - UI - Ability to make a hot-plug disk part of the VM

### PR DESCRIPTION
**Pull request by danielclowers**
**Version(s):**
4.13

**Issue:**
https://issues.redhat.com/browse/CNV-25574

**Link to docs preview:**
https://58016--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-web-new

**QE review:**
- [x] QE has approved this change.

### CODE REVIEW

**Review by lkladnit — APPROVED** [review]

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -94,7 +94,7 @@ The SVVP Certification applies to:
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
 
 //CNV-25574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM
-
+* The option to make a hot-plug volume a persistent volume has been added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```
Comment (lahinson) [inline virt/virt-4-13-release-notes.adoc:97]:
```suggestion
* The option to make a hot-plug volume a persistent volume is added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```
Use simple present tense in most cases. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense.

**Review by lahinson — COMMENTED** [review]
Just one tiny suggestion. Otherwise, LGTM!

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (danielclowers) [reply virt/virt-4-13-release-notes.adoc]: Fixed. Thanks!

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -94,7 +94,7 @@ The SVVP Certification applies to:
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
 
 //CNV-25574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM
-
+* The option to make a hot-plug volume a persistent volume is added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```
Comment (maxwelldb) [inline virt/virt-4-13-release-notes.adoc:97]: This should be .adoc rather than .html, right?

**Review by maxwelldb — COMMENTED** [review]
Check that link extension and then relabel when ready. Thanks!

**Comment by ousleyp** [comment]
LGTM! Merging.

**Pull request merged**